app/models/answer.rb

```diff
@@ -9,12 +9,16 @@
 #  text         :text
 #  created_at   :datetime
 #  updated_at   :datetime
+#  label_id     :string
```

Comment:
Not sure where this is coming from.
I didn't see the change in the schema or the migrations added.

Reply:
hmm. me either. I only have 6 records in my DB with a value for it (all pre-2014). My data model must be out of sync for that table. Will remove it in the `identifier-refactor` branch